rero_ils/jsonschemas/common/currency-v0.0.1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "currency": {
+    "title": "Currency",
+    "type": "string",
+    "enum": [
+      "CHF",
```

Comment:
Do we have to accept only this currencies?

Reply:
At this time, there are no PO demand to add other currency. By the way, it's now easy and centralized to add a currency